test.ini

```diff
@@ -9,7 +9,7 @@ error_email_from = paste@localhost
 use = config:../../ckan/test-core.ini
 ckan.site_title = My Test CKAN Site
 ckan.site_description = A test site for testing my CKAN extension
-ckan.plugins = harvest geodatagov geodatagov_miscs z3950_harvester arcgis_harvester geodatagov_geoportal_harvester waf_harvester_collection geodatagov_csw_harvester geodatagov_doc_harvester geodatagov_waf_harvester spatial_query datajson_harvest
+ckan.plugins = datagov_harvest geodatagov geodatagov_miscs z3950_harvester arcgis_harvester geodatagov_geoportal_harvester waf_harvester_collection geodatagov_csw_harvester geodatagov_doc_harvester geodatagov_waf_harvester spatial_query datajson_harvest
```

Comment:
can we import harvest and datagov_harvest, why do we need to replace it?

Reply:
You can try it locally. You will have and error. You can't have both.
`datagov_harvest` just inherit from `harvest` and add a form. 
If you activate both you will have a duplicate plugin error.